pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
@jazzido it looks like we need ghostscript as well, which can be installed with `sudo apt-get install ghostscript`. That should be added to the `.travis.yml` file the same way you added the tesseract dependency I believe.